[2024-06-06T04:07:09.167+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T04:07:09.257+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_mysql_connection manual__2024-06-06T04:07:05.137342+00:00 [queued]>
[2024-06-06T04:07:09.285+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_testConnection_v1.test_mysql_connection manual__2024-06-06T04:07:05.137342+00:00 [queued]>
[2024-06-06T04:07:09.287+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-06T04:07:09.339+0000] {taskinstance.py:2330} INFO - Executing <Task(MySqlOperator): test_mysql_connection> on 2024-06-06 04:07:05.137342+00:00
[2024-06-06T04:07:09.360+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2298) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-06T04:07:09.367+0000] {standard_task_runner.py:63} INFO - Started process 2305 to run task
[2024-06-06T04:07:09.370+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_testConnection_v1', 'test_mysql_connection', 'manual__2024-06-06T04:07:05.137342+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/TestConnectionDag.py', '--cfg-path', '/tmp/tmpqf0ysdp3']
[2024-06-06T04:07:09.380+0000] {standard_task_runner.py:91} INFO - Job 303: Subtask test_mysql_connection
[2024-06-06T04:07:09.700+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_testConnection_v1.test_mysql_connection manual__2024-06-06T04:07:05.137342+00:00 [running]> on host fe4af7f6193a
[2024-06-06T04:07:10.085+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_testConnection_v1' AIRFLOW_CTX_TASK_ID='test_mysql_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-06-06T04:07:05.137342+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-06T04:07:05.137342+00:00'
[2024-06-06T04:07:10.087+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T04:07:10.117+0000] {sql.py:276} INFO - Executing: select * from dest.pokedex
[2024-06-06T04:07:10.147+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2024-06-06T04:07:10.164+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2024-06-06T04:07:10.185+0000] {sql.py:470} INFO - Running statement: select * from dest.pokedex, parameters: None
[2024-06-06T04:07:10.194+0000] {sql.py:479} INFO - Rows affected: 1
[2024-06-06T04:07:10.200+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T04:07:10.290+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_testConnection_v1, task_id=test_mysql_connection, run_id=manual__2024-06-06T04:07:05.137342+00:00, execution_date=20240606T040705, start_date=20240606T040709, end_date=20240606T040710
[2024-06-06T04:07:10.401+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T04:07:10.479+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T04:07:10.503+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
